{{cookiecutter.project_slug}}/.github/workflows/docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Documentation
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+      - name: Cache tox
```

Comment:
I'd quite like gaps between steps. More readable and consistent with other GitHub Actions files in this repo.

Reply:
Have added spaces for the sake of consistency, though personally I find it less readable as to me it makes it harder to see how things are blocked together from the common identation level

Reply:
Happy for it to be changed - but ideally all files should look the same for users who've never encountered these before.